enumfields/drf/fields.py

```diff
@@ -19,17 +19,11 @@ def __init__(self, enum, lenient=False, ints_as_names=False, **kwargs):
         super().__init__(**kwargs)
 
     def to_representation(self, instance):
-        if instance in ('', None):
-            return instance
-        try:
-            if not isinstance(instance, self.enum):
-                instance = self.enum(instance)  # Try to cast it
-            if self.ints_as_names and isinstance(instance.value, int):
-                # If the enum value is an int, assume the name is more representative
-                return instance.name.lower()
-            return instance.value
-        except ValueError:
-            raise ValueError('Invalid value [{!r}] of enum {}'.format(instance, self.enum.__name__))
+        assert isinstance(instance, self.enum), instance
```

Comment:
Are you sure this implementation is equivalent to the old one esp. re the falsy value handling?

Reply:
Yes, the default `Serializer.to_representation` method [skips fields with None value](https://github.com/encode/django-rest-framework/blob/master/rest_framework/serializers.py#L506). As for empty strings, they are converted to None earlier by `to_python`.

Reply:
Alright. This does change behavior if one uses this field directly, but... I guess it should be alright. This will inevitably be a "possibly incompatible" version bump anyway.